src/Shared/Recruit.Vacancies.Client/Infrastructure/ReferenceData/Wages/NationalMinimumWageProvider.cs

```diff
@@ -2,24 +2,24 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Esfa.Recruit.Vacancies.Client.Application.Cache;
-using Esfa.Recruit.Vacancies.Client.Application.Configuration;
 using Esfa.Recruit.Vacancies.Client.Application.Providers;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Microsoft.Extensions.Logging;
 using MinWageEntity = Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.Wages.MinimumWage;
 
 namespace Esfa.Recruit.Vacancies.Client.Infrastructure.ReferenceData.Wages
 {
+    using System.Collections.Immutable;
```

Comment:
Can we move this class to Esfa.Recruit.Vacancies.Client.Infrastructure.Services.Wages

Reply:
Done